resources/app/src/router/people.js

```diff
@@ -72,6 +73,27 @@ export default [
         path: '/onboarding/profile',
         component: () => import (/* onboarding */ '@/views/OnboardingProfileForm.vue')
     },
+
+    {
+        name: 'RequiredDemographicsUpdateForm',
+        path: '/required-demographics-update',
+        component: () => import (/* required-demographics-update */ '@/views/RequiredDemographicsUpdateForm'),
+        meta: {
+            protected: true
+        },
+    },
+
+    {   name: 'DemographicsForm',
+        path: '/demographics/:uuid',
+        props: true,
+        //component: DemographicsForm
+       // component: DemographicsForm
+       component: () =>
+     import ( /* webpackChunkName: "application-summary" */ '@/components/people/DemographicsForm.vue'),
```

Comment:
again, the webpack magic comment should be consistent with the code it is referring to.